pkgs/top-level/all-packages.nix

```diff
@@ -17327,7 +17327,10 @@ with pkgs;
 
   syncplay = callPackage ../applications/networking/syncplay { };
 
-  syncthing = callPackage ../applications/networking/syncthing { };
+  inherit (callPackages ../applications/networking/syncthing { })
+    syncthing
+    syncthing-discovery
+    syncthing-relay;
```

Comment:
Please also create ```syncthing-cli``` with ```stcli``` inside.

Reply:
Done!  It needed a small patch since it wasn't a build target by default.